.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm run-script build
```

Comment:
what the point of that step if the build results already stored in repo?

Reply:
ok, I see, you removed build, shouldn't we run some test, does original lib have tests?

Reply:
"test": "echo \"Error: no test specified\" && exit 1"